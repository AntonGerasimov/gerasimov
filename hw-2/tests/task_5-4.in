1 2 3
4.56 3.44 0.5
