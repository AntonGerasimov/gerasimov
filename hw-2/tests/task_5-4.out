6
8.500000
